.github/workflows/e2e-linux.yml

```diff
@@ -73,6 +89,14 @@ jobs:
         restore-keys: |
           wallet-db3-${{ runner.os }}-${{ env.NETWORK }}-
 
+    - name: Download Buildkite Artifacts
+      uses: EnricoMi/download-buildkite-artifact-action@v1
+      with:
+        buildkite_token: ${{ secrets.BUILDKITE_TOKEN_READ_BUILDS_ARTIFACTS }}
+        buildkite_build_url: ${{ steps.build.outputs.url }}
+        ignore_build_states: blocked,canceled,skipped,not_run
+        output_path: ${{ env.TESTS_E2E_BINDIR }}
```

Comment:
It looks like the action does not respect the setting `working-directory: ./test/e2e`, therefore the artifacts are placed in `../../artifacts` directory. Probably we need to do something like `output_path: ${{ "./test/e2e/" + env.TESTS_E2E_BINDIR }} here. Also the downloaded artifacts are a bit unexpectedly named:

```
ls -la ../../artifacts
drwxr-xr-x  4 root   root   4096 Dec 12 15:52 .
drwxr-xr-x 15 runner docker 4096 Dec 12 15:52 ..
drwxr-xr-x  3 root   root   4096 Dec 12 15:52 Build-cardano-node-run-1
drwxr-xr-x  3 root   root   4096 Dec 12 15:52 Build-cardano-wallet-run-1
```

Reply:
So we agreed to change our approach: we wont use GA action plugins to trigger/download artifacts, instead we'll always upload artifacts as a part of the default pipeline and use Ruby to download them.